src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -250,16 +250,28 @@ interface VmNetworkTabProps {
     onAddErrorNotification: (notification: Notification) => void;
 }
 
-interface IpAddress {
+enum IpAddressVersion {
+    v4 = 0,
+    v6 = 0
+}
+
+interface IpAddressType<T extends string> {
+    type: T,
```

Comment:
I think `type: "inet" | "inet6";` should be enough. No need for the generic. Furthermore, we can use `type: IpAddressVersion`, no?

Reply:
Fixed it now!